packages/components/avatar/stories/avatar.stories.tsx

```diff
@@ -77,3 +78,37 @@ export const Cutout: Story = {
     )
   },
 }
+
+export const Group: Story = {
+  render: () => {
+    return (
+      <div className="space-y-3">
+        {['xs', 'sm', 'base', 'lg'].map((size) => (
+          <div className="gap-3 flex flex-wrap" key={size}>
+            <AvatarGroup size={size as AvatarGroupProps['size']}>
+              {Array(1).fill(<Avatar src="https://mochi.gg/logo.png" />)}
```

Comment:
If there is only one avatar, return the `Avatar` component instead of having a single avatar in the group

<img width="132" alt="image" src="https://github.com/consolelabs/mochi-ui/assets/12707960/220269aa-d3b2-45b4-80c8-ce7aeb6b4d3a">